src/examples/src/simple-component/App/template.html

```diff
@@ -1,13 +1,7 @@
 <ol>
   <!--
-    We are providing each todo-item with the todo object
-    it's representing, so that its content can be dynamic.
-    We also need to provide each component with a "key",
-    which is explained in the guide section on v-for.
+    我们给每个　todo　项提供它所表示的　todo　对象，以便能够动态展示内容。
+    同时还需要给每个组件提供一个　"key"，这在指南的　v-for　部分有详细解释。
   -->
-  <TodoItem
-    v-for="item in groceryList"
-    :todo="item"
-    :key="item.id"
-  ></TodoItem>
-</ol>
+  <TodoItem v-for="item in groceryList" :todo="item" :key="item.id"></TodoItem>
```

Comment:
保持一致的行号 [source](https://github.com/vuejs-translations/docs-zh-cn/wiki/%E7%BF%BB%E8%AF%91%E9%A1%BB%E7%9F%A5#:~:text=%E7%94%B1%E4%BA%8E%20Vue%20%E6%96%87%E6%A1%A3%E4%BB%A5%20Markdown%20%E4%B9%A6%E5%86%99%EF%BC%8C%E6%AF%8F%E4%B8%80%E8%A1%8C%E6%88%90%E4%B8%80%E4%B8%AA%E8%87%AA%E7%84%B6%E6%AE%B5%E3%80%82%E5%9B%A0%E6%AD%A4%E5%9C%A8%20Markdown%20%E6%96%87%E6%A1%A3%E4%B8%AD%E5%8E%9F%E5%88%99%E4%B8%8A%E5%BA%94%E8%AF%A5%E4%BF%9D%E8%AF%81%E4%B8%AD%E8%8B%B1%E6%96%87%E8%A1%8C%E5%8F%B7%E4%B8%80%E4%B8%80%E5%AF%B9%E5%BA%94%EF%BC%8C%E4%BB%A5%E4%BF%9D%E8%AF%81%E5%90%8E%E7%BB%AD%E6%9B%B4%E6%96%B0%E6%97%B6%E4%BD%8D%E7%BD%AE%E4%B8%8D%E5%8F%91%E7%94%9F%E9%94%99%E4%B9%B1%E3%80%82)

```suggestion
  <TodoItem
    v-for="item in groceryList"
    :todo="item"
    :key="item.id"
  ></TodoItem>
```